apiserver/facades/agent/machine/machiner.go

```diff
@@ -144,31 +160,23 @@ func (api *MachinerAPI) SetMachineAddresses(ctx context.Context, args params.Set
 	return results, nil
 }
 
-// Jobs returns the jobs assigned to the given entities.
-func (api *MachinerAPI) Jobs(ctx context.Context, args params.Entities) (params.JobsResults, error) {
-	result := params.JobsResults{
-		Results: make([]params.JobsResult, len(args.Entities)),
-	}
+// SetMachineAddresses is not supported in MachinerAPI at version 5.
+func (api *MachinerAPIv5) SetMachineAddresses(ctx context.Context, args params.SetMachinesAddresses) (params.ErrorResults, error) {
+	return params.ErrorResults{}, errors.NotSupported
+}
 
-	canRead, err := api.getCanRead()
-	if err != nil {
-		return result, err
-	}
+// Jobs is not supported in MachinerAPI at version 5.
+func (api *MachinerAPIv5) Jobs(ctx context.Context, args params.Entities) (params.JobsResults, error) {
+	return params.JobsResults{}, errors.NotSupported
+}
 
-	for i, agent := range args.Entities {
-		machine, err := api.getMachine(agent.Tag, canRead)
-		if err != nil {
-			result.Results[i].Error = apiservererrors.ServerError(err)
-			continue
-		}
-		machineJobs := machine.Jobs()
```

Comment:
Could you already remove `Jobs()` from state/machine? I'm guessing it shouldn't be used anywhere else.

Reply:
So this is only for the `Jobs()` api call. There's still a bunch of other spots that call the state/Jobs(), and to remove them we need to wire up the machine service every one of those spots, which would be beyond the scope of this PR (which is about thinning out the machiner api). But you're right, eventually all the state/Jobs() calls are gonna be going away.